src/business_logic/transaction/objects/internal_declare.rs

```diff
@@ -59,18 +59,22 @@ impl InternalDeclare {
         version: u64,
         signature: Vec<Felt252>,
         nonce: Felt252,
+        hash_value: Option<Felt252>,
```

Comment:
Why is it `BigUint` in one place but `Felt252` in another place?

Reply:
one is for Pyo3 interface, the other is the common rust interface